ssh/server/server.go

```diff
@@ -66,7 +67,7 @@ func NewServer(opts *Options, tunnel *httptunnel.Tunnel, cache cache.Cache) *Ser
 			logger.Info("new connection established")
 
 			message := func(msg string) string {
-				return fmt.Sprintf("%s\r\n", msg)
+				return fmt.Sprintf("%s\r\n", strings.ReplaceAll(msg, "\n", "\r\n"))
```

Comment:
Add a comment explaining that this is necessary due to this specific RFC.

Since you are using new lines, I don't believe you need to include a carriage return on the first line.

Reply:
Done. Do you think it is OK?